.travis.yml

```diff
@@ -1,30 +1,31 @@
+dist: xenial
```

Comment:
I updated to Xenial because the version of `gobject-introspection` packed in Trusty was incompatible with the version of `PyGObject` that pip pulls with the current restrictions.

I could make it go back to manually install the os-packaged versions, but then we lose the poer of having a single source for dependency definition. I could look into how to define versions of libraries to install depending on a library version in the OS.

Since that is not trivial I preferred to update from trusty to xenial, as trusty is going out of support on April. I also didn't occur to me this was also testing the package in a specifically old version on purpose, I though it was only used to run the unit tests. Making the tests to pass more important than the environment this happened on.